homeassistant/components/daikin/sensor.py

```diff
@@ -107,6 +122,21 @@ class DaikinSensorEntityDescription(SensorEntityDescription, DaikinRequiredKeysM
         device_class=SensorDeviceClass.ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         value_func=lambda device: round(device.last_hour_heat_energy_consumption, 2),
+        enabled_default=False,
+        cross_device=False,
+    ),
+    DaikinSensorEntityDescription(
+        key=ATTR_ENERGY_TODAY,
+        name="Today's Energy Consumption",
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        value_func=lambda device: round(
+            device.today_cool_energy_consumption + device.today_heat_energy_consumption,
```

Comment:
We don't allow integrations that integrate devices or services to calculate state. Please remove this sensor.

Reply:
But this is the main reason for this PR: Have the total energy consumption of the device (and not only heat and cool separated) and be able to put this sum entity in the energy dashboard.

Reply:
Are you going to close the PR in that case? We don't allow this, as I said.

Reply:
I'm currently not getting, what is not allowed, why? Or if we talk about different things?

I wonder that in **every** integration available in HA, the integration is doing things with the values coming from the API of the device. Summing, rounding, normalizing, clustering, retuning fitting states because of criteria? 

So isn't this the case here as well? If Daikin API delivers consumption for heat and cold. And this sensor only delivers the overall consumption of the device.  What is not allowed here and why?

And more important: How to bring this valuable total consumption into HA as total increasing sensor for users, who are not templating this and to avoid that every user has to template it? Without this sensor the user cannot put the consumption of the device into the energy dashboard, can he?

Reply:
I'll move the sum to the pydaikin library 👌